internal/cli/run_command.go

```diff
@@ -82,8 +90,8 @@ func (s Service) makeRunCommand(ctx context.Context, cfg RunConfig) (RunCommand,
 			partitionResult.utilizedPartitionCount,
 		)
 		// short circuit to avoid running the entire test suite in a single partition (e.g empty partition)
-		return RunCommand{command: partitionCommand, shortCircuit: true, shortCircuitInfo: infoMessage}, nil
+		return RunCommand{commandArgs: commandArgs, shortCircuit: true, shortCircuitInfo: infoMessage}, nil
 	}
 
-	return RunCommand{command: partitionCommand, shortCircuit: false}, nil
```

Comment:
Nice one. Yeah, this gets more of the nitty gritty out of the already quite large RunSuite func. Good eye.